mesonbuild/modules/pkgconfig.py

```diff
@@ -333,23 +335,24 @@ def generate_libs_flags(libs):
                 ofile.write('Libs: {}\n'.format(' '.join(generate_libs_flags(deps.pub_libs))))
             if len(deps.priv_libs) > 0:
                 ofile.write('Libs.private: {}\n'.format(' '.join(generate_libs_flags(deps.priv_libs))))
-            ofile.write('Cflags:')
-            for h in subdirs:
-                ofile.write(' ')
-                if h == '.':
-                    ofile.write('-I${includedir}')
-                else:
-                    ofile.write(self._escape(PurePath('-I${includedir}') / h))
-            for f in deps.cflags:
-                ofile.write(' ')
-                ofile.write(self._escape(f))
-            ofile.write('\n')
+
+            def generate_compiler_flags(dirs):
```

Comment:
dirs is not used used in this function. Also, why changing this code at all?

Reply:
Honestly, I don't even remember why I changed this at this point. I'll just change it back to the original code I suppose.

Reply:
So I had a more proper look, and seems like the old code would write out a Cflags field even when there are no Cflags to be added to the PC file. So perhaps that's why I modified the code.

Reply:
> dirs is not used used in this function

It's a small nitpicking, but that has not been fixed yet.